types/react-mathquill/react-mathquill-tests.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import MathQuill, { addStyles } from 'react-mathquill';
+import { Config, MQ } from 'edtr-io__mathquill';
```

Comment:
Please use the original name of the package.

```suggestion
import { Config, MQ } from '@edtr-io/mathquill';
```